lib/IRGen/MetadataRequest.cpp

```diff
@@ -684,11 +684,6 @@ bool irgen::isNominalGenericContextTypeMetadataAccessTrivial(
     return false;
```

Comment:
I think this is too conservative. nominal.isResilient(..., ResilienceExpansion::Minimal) ignores the module argument and returns true if the module is resilient.

But if its resilient and defined in the same module, you can still pre-specialize it. The resilience means the metadata is opaque across module boundaries, not within the module itself.